docs/detections/alerts-run-osquery.asciidoc

```diff
@@ -12,40 +12,63 @@ You must complete the following to access Osquery and run searches against your
 +
-
 [role="screenshot"]
-image::images/setup-query.png[width=80%][height=80%][Shows how to set up the query]
-
-. (Optional) Expand the **Advanced** section to view or set {kibana-ref}/osquery.html#osquery-map-fields[mapped ECS fields] included in the results from the live query.
-. Click **Submit**.
+image::images/setup-query.png[width=80%][height=80%][Shows how to set up a single query]
 
+. Click **Submit**. Queries will timeout after 5 minutes if there are no responses.
 +
 TIP: To save the query for future use, click *Save for later* and define the ID,
 description, and other {kibana-ref}/osquery.html#osquery-manage-query[details].
 
-. Review the results in the table. You can also:
-** Navigate to *Discover* to dive deeper into the response.
-** Use the drag-and-drop *Lens* editor to create visualizations.
-** Click the *Timeline* button (image:images/timeline-button-osquery.png[Click markdown icon,20,20]) to investigate a single query result in Timeline or *Add to timeline investigation* to investigate all results.
+[float]
+[[osquery-results-single]]
+=== Review single query results
+
+Results for single queries are displayed in the *Results* tab. The number of agents that were queried and their responses are shown in the top right corner of the results table. Agent responses can be `Sucessful`, `Not yet responded` (pending), and `Failed`.
+
+[role="screenshot"]
+image::images/query-results.png[width=80%][height=80%][Shows query results]
+
+[float]
+[[osquery-results-pack]]
+=== Review query pack results
+
+Results for each query in the pack are displayed in the *Results* tab. Click the expand button (image:images/pack-expand-button-osquery.png[Click markdown icon,20,20]) at the far right of each query row to display query results. The number of agents that were queried and their responses are shown for each query. Agent responses are color-coded. Green is `Sucessful`, `Not yet responded` (pending) is gray, and `Failed` is red.
+
+[role="screenshot"]
+image::images/pack-query-results.png[width=80%][height=80%][Shows query results]
+
+[float]
+[[osquery-investigate]]
+=== Investigate query results
+
+From the results table, you can:
+
+* Click the *View in Discover* button (image:images/discover-button-osquery.png[Click markdown icon,20,20])  to explore the results in Discover.
+* Click the *View in Lens* button (image:images/lens-button-osquery.png[Click markdown icon,20,20])  navigate to Lens, where you can use the drag-and-drop *Lens* editor to create visualizations.
+* Click the *Timeline* button (image:images/timeline-button-osquery.png[Click markdown icon,20,20]) to investigate a single query result in Timeline or *Add to timeline investigation* to investigate all results. This option is only available for single query results
 
 +
 TIP: An `action_ID` is generated when you run an Osquery query. The `action_ID` field and value pair is passed to the Timeline's KQL filter when you select the option to open all results in Timeline.
```

Comment:
I know this isn't part of the PR proper, but it's unclear what exactly the user would do with this info. I think it gets bogged down in explaining the mechanics of an action_ID instead of focusing on its usefulness.

Here's one possibility, which I don't think needs to be highlighted as a Tip or even a Note, because it's really just basic information; it could be just another sentence added to the previous bullet/para about Timeline:
```suggestion
When you open all results in Timeline, the events in Timeline are filtered based on the `action_ID` generated by the Osquery query.
```

Reply:
I see the point you're making and your revision is correct. :)